cucumber/authenticators_jwt/features/authn_jwt_check_standard_claims.feature

```diff
@@ -0,0 +1,367 @@
+Feature: JWT Authenticator - Check standard claim
+
+  Background:
+    Given I initialize JWKS endpoint with file "myJWKs.json"
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: token-app-property
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !policy
+      id: conjur/authn-jwt/keycloak
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on Keycloak as OIDC provider.
+
+      - !variable
+        id: provider-uri
+
+      - !variable
+        id: token-app-property
+
+      - !variable
+        id: issuer
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !host
```

Comment:
I think it suppose to be user

Reply:
Copied it from here
https://github.com/cyberark/conjur/blob/7f55e7a1a104b5ae07eeba694c255df4dbe9b0c0/cucumber/authenticators_jwt/features/authn_jwt_provider_uri.feature#L33

We can discuss this

Reply:
Will change to hosts

Reply:
Changed